agent/php_execute.c

```diff
@@ -107,6 +107,44 @@ typedef void (*nr_framework_enable_fn_t)(TSRMLS_D);
  */
 typedef void (*nr_library_enable_fn_t)(TSRMLS_D);
 
+/*
```

Comment:
I wish 3e7ec823504a22e68c094b947fd86af06ebd41d9 moved to where it used to be (right before the doc string of `nr_php_execute_metadata_init`). This way diff would only show these two lines as changes done for CLM:
```
  zend_string* filepath;
  uint32_t function_lineno;
```

Reply:
Please see: [59bb8ff](https://github.com/newrelic/newrelic-php-agent/pull/585/commits/59bb8ff4132dfbb5b6efc25001a103e794752756)

Reply:
Thank you! Now it's easy to see in this PR what CLM actually changed/added.